File: openapi/code_samples/csharp/api_company_{cui}_financial_indicators_chart.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("indicators-chart/{cui}")]
    [SwaggerOperation(Summary = "Retrieves the financial indicators in a chart view of a given company CUI.")]
    [ProducesResponseType(typeof(FinancialIndicatorChartResponseDTO), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult ListIndicatorsAsChart([FromRoute] string cui, [FromQuery(Name = "currency")] string currency = "RON")
    {
        // Your implementation here
        return Ok();
    }
}

File: openapi/code_samples/python/api_company_{cui}_financial_indicators_chart.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/indicators-chart/<string:cui>', methods=['GET'])
def list_indicators_as_chart(cui):
    currency = request.args.get('currency', 'RON')
    # Your implementation here
    return jsonify({}), 200

if __name__ == '__main__':
    app.run()
